src/imitation/util/networks.py

```diff
@@ -1,31 +1,66 @@
 """Helper methods to build and run neural networks."""
 
 import collections
-from typing import Callable, Iterable, Optional
+from typing import Iterable, Optional, Type
 
 from torch import nn
 
 
+class SqueezeLayer(nn.Module):
+    """Torch module that squeezes a B*1 tensor down into a size-B vector."""
+
+    def forward(self, x):
+        assert x.ndim == 2 and x.shape[1] == 1
+        new_value = x.squeeze(1)
+        assert new_value.ndim == 1
+        return new_value
+
+
 def build_mlp(
     in_size: int,
     hid_sizes: Iterable[int],
+    out_size: int = 1,
     name: Optional[str] = None,
-    activation: Optional[Callable] = nn.ReLU,
-    initializer: Optional[Callable] = None,
+    activation: Type[nn.Module] = nn.ReLU,
+    squeeze_output=False,
 ) -> nn.Module:
-    """Constructs an MLP, returning an ordered dict of layers."""
+    """Constructs a Torch MLP.
+
+    Args:
+        in_size: size of individual input vectors; input to the MLP will be of
+            shape (batch_size, in_size).
+        hid_sizes: sizes of hidden layers.
+        out_size: required size of output vector.
+        activation: activation to apply after hidden layers.
+        squeeze_output: if out_size=1, then squeeze_input=True ensures that MLP
+            output is of size (B,) instead of (B,1).
+
+    Returns:
+        nn.Module: an MLP mapping from inputs of size (batch_size, in_size) to
+            (batch_size, out_size), unless out_size=1 and squeeze_output=True,
+            in which case the output is of size (batch_size, )."""
```

Comment:
Add a `Raises` section to docstring: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods